packages/node-api/src/LogionLoc.ts

```diff
@@ -343,7 +356,15 @@ export function addCollectionItem(parameters: AddCollectionItemParameters): Subm
         collectionId,
         itemId,
         itemDescription,
+        itemFiles,
     } = parameters;
 
-    return api.tx.logionLoc.addCollectionItem(collectionId.toHexString(), itemId, itemDescription);
+    const files: ({ name?: any; contentType?: any; size_?: any; hash_?: any })[] = itemFiles.map(itemFile => ({
+        name: stringToHex(itemFile.name),
+        contentType: stringToHex(itemFile.contentType),
+        size_: itemFile.size,
+        hash_: itemFile.hash,
+    }));
+
+    return api.tx.logionLoc.addCollectionItem(collectionId.toHexString(), itemId, stringToHex(itemDescription), files);
```

Comment:
Seems weird to me that we now convert description to Hex - Not backward compatible, and neither equivalent change in `getCollectionItem`. Did I miss something ?

Reply:
Actually, calling `stringToHex` on the string is just making explicit what happens under the hood. Indeed, the string must be converted somehow into a vector of bytes (which is the type expected by the runtime in the end). We use that approach on LOC metadata's name and value and it works as expected.